doc/PROTOCOL-HTTP2.md

```diff
@@ -229,6 +239,24 @@ CONNECT_ERROR|INTERNAL
 ENHANCE_YOUR_CALM|RESOURCE_EXHAUSTED ...with additional error detail provided by runtime to indicate that the exhausted resource is bandwidth.
 INADEQUATE_SECURITY| PERMISSION_DENIED … with additional detail indicating that permission was denied as protocol is not secure enough for call.
 
+##### Client Cancellation
```

Comment:
nit: Only the first sentence is about cancellation, with the rest covering deadlines.

While they are related, maybe there should be a separate deadline section. Or the first sentence that is specific to cancellation be its own paragraph.